packages/js/src/plugins/auctionHouseModule/updateAuctionHouse.ts

```diff
@@ -149,4 +159,50 @@ export const updateAuctionHouseBuilder = (
         key: params.instructionKey ?? 'updateAuctionHouse',
       })
     );
+
+  if (!auctionHouse.hasAuctioneer && params.auctioneerAuthority) {
+    const auctioneerAuthority = params.auctioneerAuthority
+      ?.publicKey as PublicKey;
+    const ahAuctioneerPda = findAuctioneerPda(
+      auctionHouse.address,
+      auctioneerAuthority
+    );
+
+    const ahAuctioneerAccount = await metaplex
+      .rpc()
+      .getAccount(ahAuctioneerPda);
+
+    // Skip delegate if it has been made already
+    if (ahAuctioneerAccount.exists) {
+      return builder;
+    }
+
+    const scopes = params.auctioneerScopes ?? [
+      AuthorityScope.Deposit,
+      AuthorityScope.Buy,
+      AuthorityScope.PublicBuy,
+      AuthorityScope.ExecuteSale,
+      AuthorityScope.Sell,
+      AuthorityScope.Cancel,
+      AuthorityScope.Withdraw,
+    ];
+
+    builder.add({
+      instruction: createDelegateAuctioneerInstruction(
+        {
+          auctionHouse: auctionHouse.address,
+          authority: authority.publicKey,
+          auctioneerAuthority,
+          ahAuctioneerPda,
+        },
+        {
+          scopes,
+        }
+      ),
+      signers: [payer],
```

Comment:
This should be the authority of the auction house.

```suggestion
      signers: [authority],
```